### PR TITLE
OCD-2973 - Revise list of test tools to select for a 2015 Edition listing

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: src/app/components/listing/details/criteria/edit.html [hunk 1]
```diff
@@ -196,8 +196,8 @@ <h4 class="modal-title" id="edit-criteria-label">Certification Criteria: <span n
                   <chpl-expandable-list
                     identifier="testTools"
                     item-key="id"
-                    item-text="dropDownText"
-                    items="$ctrl.resources.testTools.data"
+                    item-text="name"
```
Comment (andlar) [inline src/app/components/listing/details/criteria/edit.html]: If the display field is changed, shouldn't whatever it is that was removed get removed from the controller as well? The thing that created the `dropDownText`? Also, was there anything in that logic that should be restored?

**Review by kekey1 — DISMISSED** [review]

File: src/app/components/listing/details/criteria/edit.component.js [hunk 2]
```diff
@@ -323,11 +323,25 @@ const CertificationCriteriaEditComponent = {
...
+            this.cert.allowedTestTools.push({
+              id: tt.testToolId,
+              name: tt.testToolName,
+              retired: tt.retired,
+              dropDownText: tt.testToolName + (tt.retired ? ' (Retired)' : ''),
+            });
+          }
+        });
+      }
+      this.$log.info(this.cert.allowedTestTools);
```
Comment (andlar) [inline src/app/components/listing/details/criteria/edit.component.js]: Debugging code?

File: src/app/services/network.service.js [hunk 3]
```diff
@@ -602,10 +602,6 @@ export default class NetworkService {
     return this.apiGET('/data/test_standards');
   }
 
-  getTestTools() {
-    return this.apiGET('/data/test_tools');
```
Comment (andlar) [inline src/app/services/network.service.js:606]: Do the unit tests pass?

**Review by andlar — DISMISSED** [review]

File: src/app/components/listing/details/criteria/edit.html [hunk 4]
```diff
@@ -187,7 +187,7 @@ <h4 class="modal-title" id="edit-criteria-label">Certification Criteria: <span n
           </td>
           <td ng-if="(!$ctrl.cert.testProcedures || $ctrl.cert.testProcedures.length === 0) && !$ctrl.canEdit()">None</td>
         </tr>
-        <tr ng-if="$ctrl.cert.success && $ctrl.cert.testToolsUsed !== null">
+        <tr ng-if="$ctrl.cert.success && (($ctrl.cert.testToolsUsed && $ctrl.cert.testToolsUsed.length > 0) || ($ctrl.cert.allowedTestTools && $ctrl.cert.allowedTestTools.length >0))">
```
Comment (kekey1) [inline src/app/components/listing/details/criteria/edit.html:190]:
```suggestion
        <tr ng-if="$ctrl.cert.success && (($ctrl.cert.testToolsUsed && $ctrl.cert.testToolsUsed.length > 0) || ($ctrl.cert.allowedTestTools && $ctrl.cert.allowedTestTools.length > 0))">
```

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Pull request merged**